### PR TITLE
Ongyerth/multi region

### PR DESCRIPTION
**Pull request by Ongy**
Add handling for clusters in multiple regions for lower latency routes in global settings.

To use this, the config.sh has to contain the new parameter. with a single load balancing cluster in the EU, this is as easy as:

`LB_ENDPOINTS=('{"name": "europe-west1", "region": "europe-west1", "zone": "europe-west1-b"}')`

### CODE REVIEW

**Comment by ensonic** [comment]
Awesome! Can you share (e.g. gist) how the modified config.sh looks like that defines the LB setup?

**Comment by csieber** [comment]
Cool change! I am just not sure if in the context of cloud robotics this can be called load balancing. Do you think maybe additional regions would be a better naming?

**Review by csieber — APPROVED** [review]

File: deploy.sh [hunk 1]
```diff
@@ -340,6 +359,64 @@ function helm_cleanup {
   cleanup_old_cert_manager
 }
 
+function helm_ar {
```
[the remark this answers is not in the document]
Comment (Ongy) [reply deploy.sh]: Done. This worked way cleaner than I thought it will.

[the remark this answers is not in the document]
Comment (csieber) [reply deploy.sh]: Looks good!

**Pull request merged**